packages/ledger-amino/src/ledgersigner.ts

```diff
@@ -37,6 +43,19 @@ export class LedgerSigner implements OfflineAminoSigner {
     return this.accounts;
   }
 
+  public async showAddress(path: HdPath): Promise<AddressAndPubkey> {
+    const response = await this.ledger.verifyAddress(path);
+
+    if (response.error_message) {
+      throw new Error(response.error_message);
+    }
+
+    return {
+      address: response.address,
+      pubkey: response.compressed_pk,
```

Comment:
This line does not compile because compressed_pk is a Buffer and pubkey is a struct